lib/model.dart

```diff
@@ -7,3 +8,17 @@ class Link {
 
   Link(this.pathname, this.label, this.render);
 }
+
+class TickOptions implements chartjs.TickOptions {
```

Comment:
I don't think it's a good idea to do this - `chartjs.TickOptions` is a wrapper class over the JS library, and so I'm not actually sure that subclassing it will work as expected? (I had the impression that the wrapper classes over JS libraries were abstract with the JS code providing the implementation, which means that your implementation of `TickOptions` here would be parallel to the JS code, and not a replacement?) Could we look instead at adding the `min` field to `chartjs.TickOptions`?

Reply:
I'm a bit lost about the dart classes over js lib, will look it up and reach out for questions separately.

I have issued the PR with the fixes to here: https://github.com/marianamarasoiu/chartjs.dart/pull/4